torchqdynamics/mesolve.py

```diff
@@ -1,14 +1,21 @@
 from math import sqrt
+from typing import List, Optional
 
 import torch
 
-from .odeint import odeint
+from .odeint import AdjointQSolver, odeint
 from .solver import Rouchon
+from .solver_utils import inv_sqrtm, kraus_map
+from .utils import trace
 
 
 def mesolve(
-    H, jump_ops, rho0, tsave, solver=None, sensitivity='autograd', parameters=None
+    H, jump_ops, rho0, tsave, exp_ops: Optional[List[torch.Tensor]] = None, solver=None,
+    sensitivity='autograd', parameters=None
 ):
+    tsave = torch.tensor(tsave)
+    if exp_ops is None:
+        exp_ops = exp_ops
```

Comment:
What does this do? Is it a typo?

Reply:
Yes I meant the following sorry, I'll fix it
```suggestion
        exp_ops = []
```